woocommerce/woocommerce-gutenberg-products-block
# Document the selectors for the wc/store/cart data store

In this PR, we're adding the document the selectors for the cart store. 

Fixes #7690 
Fixes #7451 

#### Changes in the PR:

- Fixed the readme file links. 
- Added selectors doc for cart store. 


#### Testing



1. Go to [readme file link of this branch](https://github.com/woocommerce/woocommerce-blocks/blob/fdbf366f3820e2e10934f63cc582ae7ed7a0fbdc/docs/third-party-developers/extensibility/data-store/README.md).
2. Confirm the link to payment doc is working fine. 
3. Confirm the link to Cart doc is working fine. 
4. Review the Cart doc. 




### Changelog

> Added documentation for the selectors of the cart data store
